celery_batches/__init__.py

```diff
@@ -304,6 +322,11 @@ def flush(self, requests):
 
         def on_accepted(pid, time_accepted):
             [req.acknowledge() for req in acks_late[False]]
+            for req in requests:
+                # Start time of the task, which will be useful in calculating runtime of the task
+                req.start_time = time_accepted
```

Comment:
I think Celery calls this `time_start`?

https://github.com/celery/celery/blob/8d6778810c5153c9e4667eed618de2d0bf72663e/celery/worker/request.py#L69